src/main/java/ch/njol/skript/tests/platform/PlatformMain.java

```diff
@@ -62,6 +65,10 @@ public static void main(String... args) throws IOException, InterruptedException
 		assert envsRoot != null;
 		boolean devMode = "true".equals(args[4]);
 		boolean genDocs = "true".equals(args[5]);
+		boolean debug = "true".equals(args[6]);
+		String verbosity = args[7].toUpperCase(Locale.ENGLISH);
+		List<String> jvmArgs = Lists.newArrayList(Arrays.copyOfRange(args, 8, args.length));
+		jvmArgs.add("-Xmx5G");
```

Comment:
You should check whether another Xmx argument has already been passed within args.